### ISSUE TITLE
Add installation and usage instructions to README.md

### ISSUE THREAD

**Issue by worldbeater** [TolikPylypchuk/KeyboardSwitch#29]
Hello @TolikPylypchuk and thanks for the great open-source application! Just tested on my Windows PC, so far works well. Browsing the source code here was a real pleasure, noticed a few curious patterns, including [`ReactiveForm`](https://github.com/TolikPylypchuk/KeyboardSwitch/blob/master/KeyboardSwitch.Settings.Core/ViewModels/ReactiveForm.cs) and localization pattern for [ReactiveUI.Validation](https://github.com/reactiveui/reactiveui.validation). However, worth adding a bit more verbose getting started documentation to README.md, which a user could easily follow to get `KeyboardSwitch` up and running on their machine.

By the way, it would be really nice to have your application listed on the [Samples page on the ReactiveUI website](https://reactiveui.net/docs/resources/samples#avaloniaui). Please consider opening a PR to our [website repository (Avalonia samples section)](https://github.com/reactiveui/website/blob/master/input/docs/resources/samples.md#avaloniaui) linking the source code of the application, once you feel the app is ready to get listed there 🥰
Thank you!

**Comment by TolikPylypchuk** [comment]
Hi @worldbeater ! Thanks for your feedback and your interest in the app!

I agree, docs are a bit lacking at the moment, because I thought I needed to finalize version 3 itself before writing them. Older versions include more detailed instructions, and I'm planning on writing those for version 3 as well before I release it. Version 3 will include an installer as well, which will configure it to run at startup. I'm also actively looking into auto-configuration, so that users don't need to manually map every character after installation.

To be honest, I never thought anyone besides my friends would use this app :) But once version 3 is ready, I'm definitely going to open a PR to include it into the ReactiveUI samples, thanks for the suggestion!

**Comment by TolikPylypchuk** [comment]
Now that there are extensive docs for the app ([which can be found here](https://tolikpylypchuk.github.io/KeyboardSwitch/)) I can finally close this issue :)
